src/main/java/org/mitre/dsmiley/httpproxy/ProxyServlet.java

```diff
@@ -131,7 +137,44 @@ protected HttpHost getTargetHost(HttpServletRequest servletRequest) {
    * it can be overridden.
```

Comment:
I think these javadocs are now missing info about the `${}` syntax you added and properties files.